src/test/java/net/sf/jabref/event/ChangeEntryEventTest.java

```diff
@@ -22,11 +23,11 @@ public void setUp() {
     @Test
     public void testEventReceivement() {
         BibEntry shouldBeBibEntry = new BibEntry();
-        shouldBeBibEntry.setId("testkey2");
+        shouldBeBibEntry.setId(IdGenerator.next());
```

Comment:
When constructing an entry using the default constructor, `IdGenerator.next()` is already called. No need to do an additional call.